src/pages/settings/Security/TwoFactorAuth/TwoFactorAuthSteps.tsx

```diff
@@ -1,50 +1,55 @@
-import {useRoute} from '@react-navigation/native';
+import { useRoute } from '@react-navigation/native';
+// eslint-disable-next-line
 import lodashGet from 'lodash/get';
```

Comment:
dont use `lodashGet` use optional chaining where its needed

Reply:
@GandalfGwaihir you need to type `useRoute` here is a link which can help you https://github.com/react-navigation/react-navigation/issues/7663#issuecomment-1048324363